src/sitemap.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url><loc>http://dex.software/home</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/project/overview</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/project/add/source</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/account/login</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/privacy</loc><priority>0.5</priority></url>
```

Comment:
```suggestion
```
I don't think that the privacy policy is relevant for a sitemap. I leave it out on all my own sites. Search engines also decide what's important as well.

Also, if you were to leave it in, the URL's incorrect.